CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 0.8.0-rc.0
```

Comment:
Add directly `0.8.0` (without the `-rc.0`) since you're merging with the default branch (do not care about marketing, we solved that in a different way that does not change our workflow)